client/app/components/mails_input.coffee

```diff
@@ -159,7 +159,7 @@ module.exports = MailsInput = React.createClass
                 "#{contact.get 'fn'} <#{contact.get 'address'}>"
 
     onQuery: (char) ->
-        query = @refs.contactInput.getDOMNode().value.split(',').pop().trim()
+        query = @refs.contactInput.getDOMNode().value.split(',').pop().trimLeft()
```

Comment:
No, you're right, it works in Firefox and Chromium but may fail in other engines, I'm gonna fix it.